pkgs/data/themes/base16-schemes/default.nix

```diff
@@ -7,8 +7,8 @@ stdenv.mkDerivation (finalAttrs: {
   src = fetchFromGitHub {
     owner = "tinted-theming";
     repo = "schemes";
-    rev = "395074124283df993571f2abb9c713f413b76e6e";
```

Comment:
This is not the version, but the `version =` line above.

Reply:
The package doesn't have versions, I just picked a commit from the master branch.

Reply:
When the package has no version, we fill the `version` field with a string like `0-unstable-YYYY-MM-DD`, where the date is the same date when the commit you extracted was made.

Reply:
OK.